docs/Dependencies.md

```diff
@@ -1,5 +1,6 @@
 # Dependencies
 * cmake
+* nightly-build
```

Comment:
What's this about? `apt-cache show nightly-build` doesn't return anything on my system (ubuntu specific?).

Reply:
Yea, was the wrong one, it was ninja-build. Don't know how nightly came in my mind.